cmd/acr/purge.go

```diff
@@ -264,6 +270,21 @@ func getTagsToDelete(ctx context.Context,
 				// If a tag does not match the regex then it not added to the list no matter the LastUpdateTime
 				continue
 			}
+			isExcluded := false
+			for _, exFilter := range excludeFilter {
+				exFilterRegex, exFilterErr := regexp.Compile(exFilter)
+				if exFilterErr != nil {
+					return nil, "", exFilterErr
+				}
+				if exFilterRegex.MatchString(*tag.Name) {
+					isExcluded = true
+					continue
+				}
+			}
+			if isExcluded {
```

Comment:
Once the `excludeFilter` is a `*regexp.Regexp`, the variable `isExcluded` can be simplified.